_posts/2019-08-23-hobby-apps.md

```diff
@@ -39,7 +39,7 @@ If you can answer the question of _why_ create a hobby app, the answer to the qu
 
 2. Some app ideas present difficult logistical challenges. For example, I might have a great idea for a social-networking app and the chops to implement the app and backend, but I do not have the time or budget to convince people to use a new social-networking app. This lack of users would frustrate the purpose of a social-networking app, connecting with other humans. Because of these logistical challenges, I would not create a social-networking app.
 
-3. I sometimes compose and edit long-form text on my iPhone, primarily in Notes. I could create a text-editing app. Relevant APIs spring to mind: `UITextView`, CoreData, and CloudKit. But I have no ideas for features I could include in a text editor that existing apps like Notes, Bear, and Google Docs do not already provide. In marketing terms, I would be unable to articulate a unique value proposition for my hypothetical text-editing app. In the absence of a unique value proposition, I would not be motivated to maintain the app for my own use or advocate use of the app to others. The app would likely languish in the App Store, unupdated, until Apple pulled it after some form-factor, OS, or architecture change. Given the effort involved in creating and releasing an app, this outcome would sadden me. Accordingly, I would not create a text-editing app.
+3. I sometimes compose and edit long-form text on my iPhone, primarily in Notes. I could create a text-editing app. Relevant APIs spring to mind: `UITextView`, CoreData, and CloudKit. But I have no ideas for features I could include in a text editor that existing apps like Notes, Bear, and Google Docs do not already provide. In marketing terms, I would be unable to articulate a unique value proposition for my hypothetical text-editing app. In the absence of a unique value proposition, I would not be motivated to maintain the app for my own use or advocate use of the app to others. The app would likely languish in the App Store, un-updated, until Apple pulled it after some form-factor, OS, or architecture change. Given the effort involved in creating and releasing an app, this outcome would sadden me. Accordingly, I would not create a text-editing app.
```

Comment:
Interestingly, Atom, the text editor I used, also flagged `unupdated` as a misspelling. But at least one [dictionary](https://en.wiktionary.org/wiki/unupdated) has this spelling, which I prefer from an esthetic standpoint.